tests/FSharp.Compiler.ComponentTests/Conformance/InferenceProcedures/ByrefSafetyAnalysis/ByrefSafetyAnalysis.fs

```diff
@@ -8,6 +8,215 @@ open FSharp.Test.Compiler
 
 module ByrefSafetyAnalysis =
 
+    // SOURCE=MigratedTest01.fs SCFLAGS="--test:ErrorRanges"                              # MigratedTest01.fs
+    [<Theory; Directory(__SOURCE_DIRECTORY__, Includes=[|"MigratedTest01.fs"|])>]
+    let``MigratedTest01_fs`` compilation =
+        compilation
+        |> ignoreWarnings
```

Comment:
Aren't those missing `--test:ErrorRanges`, or is it not required anymore?

Reply:
What does that option do?

Reply:
Yeah, you can pass it like it.

Not sure what it does though. It might emit additional output (like many --test: options do).

Will need to look at it.

Reply:
Won't be surprised it doesn't do anything anymore though.

Reply:
Yeah, it's not needed much in new style tests.